src/picongpu/submit/hypnos-hzdr/picongpu.profile.example

```diff
@@ -45,6 +45,8 @@ export PATH=$PATH:$PICSRC
 export PATH=$PATH:$PICSRC/src/splash2txt/build
 export PATH=$PATH:$PICSRC/src/tools/bin
 
+export PYTHONPATH=$PICSRC/src/tools/lib/python:$PYTHONPATH
```

Comment:
@ComputationalRadiationPhysics/picongpu-developers please add this line to your `picongpu.profile` if you are working on `dev`